workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -7,13 +6,25 @@ namespace Improbable.Gdk.Tools
 {
     public static class DownloadCoreSdk
     {
-        [MenuItem("Improbable/Download CoreSdk")]
+        [MenuItem("Improbable/Download CoreSdk", false, 50)]
```

Comment:
`private const int DownloadCoreSdkMenuPriority = 50` and similar?

Reply:
Sure. The whole "Space menu item priorities by > 10 to inject a menu separator " thing is crazy!